x/staking/keeper/grpc_query.go

```diff
@@ -92,33 +92,33 @@ func (k Querier) ValidatorDelegations(c context.Context, req *types.QueryValidat
 	if req.ValidatorAddr == "" {
 		return nil, status.Error(codes.InvalidArgument, "validator address cannot be empty")
 	}
+
```

Comment:
should we have a way for historical heights to still use the old key? we did something like this in bank after crypto.com asked